.github/workflows/test.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       matrix:
         python: ['3.8']
         django: ['2.2', '3.1']
-        wagtail: ['2.11', '2.12', '2.13']
+        wagtail: ['2.11', '2.12', '2.13', '2.14', '2.15']
```

Comment:
Wagtail 2.15 requires Django >= 3.0 - https://github.com/wagtail/wagtail/blob/stable/2.15.x/setup.py#L23. with this you will get a test failure.

here's how to add additional env combinations outside of the matrix:
https://github.com/wagtail/wagtail-localize/blob/main/.github/workflows/test.yml#L76-L79